.github/workflows/ci.yml

```diff
@@ -47,8 +50,10 @@ jobs:
           key: ${{ runner.os }}-build-${{ github.sha }}
 
       - name: Install dependencies
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        run: yarn --ignore-scripts --frozen-lockfile
+        run: yarn --immutable
+
+      - name: Perform a yarn install
+        run: yarn install
```

Comment:
question: Are you sure this is needed, when `yarn --immutable` is run just above?

Reply:
Will discuss on slack - will try removing but I think it is...